tanner/session_analyzer.py

```diff
@@ -131,3 +137,18 @@ def choose_possible_owner(stats):
         maxval = max(possible_owners.items(), key=operator.itemgetter(1))[1]
         owners = [k for k, v in possible_owners.items() if v == maxval]
         return {'possible_owners': owners}
+
+    @staticmethod
+    def find_location(ip):
```

Comment:
why you decided to use synchronous urlopen? why not async function with aiohttp request?

Reply:
And the most important: "__deprecation_message__":"This API endpoint is deprecated and will stop working on July 1st, 2018. For more information please visit: https://github.com/apilayer/freegeoip#readme"